UnityProject/Assets/Scripts/Items/Implants/Organs/Brain.cs

```diff
@@ -48,11 +50,18 @@ public override void SetUpSystems()
 		public override void AddedToBody(LivingHealthMasterBase livingHealth)
 		{
 			livingHealth.Setbrain(this);
-			if (CannotSpeak)
+			if (CannotSpeak || hasInbuiltSpeech)
```

Comment:
this looks weird. Maybe just make them two consecutive if's. First check ``CanNotSpeak`` then ``hasInBuiltSpeech``. No nesting inside and if trying them both.